packages/api/src/email-campaign/dto/email-campaign-input.factory.ts

```diff
@@ -32,6 +33,11 @@ export class EmailCampaignInputFactory {
             @Field(() => Date, { nullable: true })
             scheduledAt?: Date;
 
+            @IsNullable()
+            @Field(() => ID, { nullable: true, defaultValue: null })
```

Comment:
why is there a defaultValue: null?
And why is the IsNullable validator used here and not IsUndefinable ?

Reply:
I removed the default value and added the IsUndefinable validator in [7cc2c25](https://github.com/vivid-planet/comet-brevo-module/pull/21/commits/7cc2c250c8812dabb145da3bfa80bd62cbc07f61)